plugins/europeana/entity.js

```diff
@@ -258,6 +258,28 @@ export function getEntityDescription(entity) {
   return description;
 }
 
+/**
+ * A check for a URI to see if it conforms ot the entity URI pattern
+ * Will return true/false
+ * @param {String} A URI to check
+ * @return {Boolean} true if the URI is a valid entity URI
+ */
+export function isEntityUri(uri) {
+  return !!uri.match(/^http:\/\/data\.europeana\.eu\/(concept|agent|place)\/base\/\d{1,9}$/);
+}
+
+/**
+ * From a URI split params as required by the portal
+ * @param {String} A URI to check
+ * @return {{type: String, identifier: string}} Object with the portal relevant identifiers.
+ */
+export function entityParamsFromUri(uri) {
+  const matched = uri.match(/^http:\/\/data\.europeana\.eu\/(concept|agent|place)\/base\/(\d{1,9})$/);
```

Comment:
Why is the numeric ID limited to 9 digits?

Reply:
no reason other than it didn't seem like there would be entity IDs that high, I've however changed it to alow any number of digits.